README.md

```diff
@@ -223,6 +246,12 @@ proxy_set_header Host <IdP hostname>;
 proxy_ssl_name        <IdP hostname>;
 ```
 
+  * **Invalid access token**
+    Users may receive the response with HTTP 401 "invalid token" despite of successful authentication. There are several reasons why an OIDC access token might be not accepted by upstream although if it is not expired as follows:
+    * Incorrect backend server configuration. NGINX Plus sends bearer token in the HTTP Authorization header, but the backend application expects it in a specific cookie.
+    * The token has been tampered with. OIDC access tokens are digitally signed by the authorization server to ensure their authenticity. If the token has been modified in any way, the signature will no longer be valid, and the token will be considered invalid.
+    * It is important to note that the scope of an OIDC access token is independent of its validity. Even if an OIDC access token is not expired and has not been revoked, it may still be considered invalid if it does not have the necessary scope for the requested action. Please check "$oidc_scopes" variable in the "openid_connect_configuration.conf" file.
```

Comment:
```suggestion
    > **Note:** The scope of an OIDC access token is independent of its validity. Even if an OIDC access token is not expired and has not been revoked, it may still be considered invalid if it does not have the necessary scope for the requested action. Please check the `$oidc_scopes` variable in the `openid_connect_configuration.conf` file.
```

Reply:
Thanks! Revised!